modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtInstancelevel.testcase

```diff
@@ -133,9 +133,17 @@ definition {
 
 		task ("And Given 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File") {
 			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+
+			ImportExport.configureExport(
+				entityType = "C_Student (v1_0 - Liferay Object REST)",
+				exportFileFormat = "JSON");
 		}
 
 		task ("When exporting file in JSON format with all fields selected") {
+			ScrollWebElementIntoView(
```

Comment:
with scroll-into-view the button is active, no need to interact with other elements